README.md

```diff
@@ -45,6 +45,33 @@ Ex. If you are literally running build.sh locally..
 ```bash
 $ docker-machine create --driver virtualbox --virtualbox-memory "4096" dev
 ```
+
+## Connect to Cassandra cluster
+
+When you want to use the docker images not only for development but also for production
```

Comment:
I think should tighten up this whole section, I think we can reduce character count significantly and avoid discussion about what is production or not.

How about this?
---
## Connecting to Cassandra for Span Storage

`zipkin-collector` and `zipkin-query` store and retrieve spans from Cassandra, using its native protocol. Specify a list of one or more Cassandra nodes listening on port 9042, via the comma-separated environment variable `CASSANDRA_SEEDS`.

ex. 
```bash
docker run ... --name="zipkin-query" -e "CASSANDRA_SEEDS=node1,node2,node3" ...
```
---

We could also tighten up and use the compose.yaml as an example for local/laptop development (via just linking to it)